src/background.js

```diff
@@ -322,7 +322,29 @@ function stripFbclid(url) {
   return strippedUrl.href;
 }
 
+async function tabUpdateListener (tabId, changeInfo, tab) {
+  await updateBrowserActionIcon(tab.url);
+}
+
+async function updateBrowserActionIcon (url) {
+  if(isFacebookURL(url)) {
+    browser.browserAction.setPopup({popup: "./panel1.html"});
+    const FBC_STORAGE = await browser.storage.local.get();
+    if (FBC_STORAGE.PANEL_SHOWN !== true) {
+      await browser.browserAction.setBadgeBackgroundColor({
+        color: "#3B5998"
+      });
+      browser.browserAction.setBadgeText({text: "funny"});
```

Comment:
Change the text to a real translation string.

Reply:
I wasn't sure which string to add here replacement of this. So except this, all the requested changes are done. :)

Reply:
Oops, yeah I was trying different text strings to see how much of each would show up in the badge.